### ISSUE TITLE
Journal-Einträge sind da, Karte nicht

### ISSUE THREAD

**Issue by Angband666** [himmelweiss/dsa5-aventuria-map#9]
Die Initialisierung des Moduls hat geklappt ohne Fehlermeldung. Leider sehe ich nur einen grauen Canvas-Hintergrund mit vielen Journal-Icons. Die Icons sind allerdings alles die Standardicons (Buchsymbol). Die eigentliche Karte kann ich nicht sehen. In meinem "Data"-Ordner konnte ich auch gar keine Karten-Dateien finden. Woran könnte es liegen?

**Comment by JuergenStein** [comment]
Die eigentlichen Karten kommen direkt von dsa5-core, also vom DSA5 System. Hier sicher stellen das die neueste Version am laufen ist.
Ansonsten evtl. die Karte in der Szene neu zuweisen (_hires).

Es könnte aber auch sein, das die "Max Texture Size" der Grafikkarte zu gering ist und deshalb die _hires Karte nicht geladen werden kann und dir die Szene daher grau oder schwarz angezeigt wird.
Wenn dem so sein sollte dann die normale lowres Karte stattdessen als Hintergrundbild festlegen in der Szene.
Die lowres hat zwar einen leichten Offset wodurch dann die Map Notes leicht verschoben sind, also nicht ganz an der richtigen Stelle, aber im groben sollte das dennoch passen.

Das die Icons in deinem Fall nur Standardicons sind ist mir allerdings neu, hier weiß ich leider aktuell keinen Rat.

**Comment by Angband666** [comment]
![Screenshot 2024-11-23 124618](https://github.com/user-attachments/assets/e8f8539b-5ba1-43fa-a607-00c171e97490)
Mein Server ist ein Synology NAS. Ich habe keine Ahnung ob die Grafikleistung davon ausreicht. Ich nutze es aber auch zum Videostreaming und da funktioniert es ruckelfrei. Ich hab mal einen Screenshot gemacht, denn ich komme nicht weiter. Es ist gar keine Grafikdatei hinterlegt ("Background Image") und in den Ordnern des Moduls finde ich auch keine dergleichen. Was mache ich falsch?

**Comment by JuergenStein** [comment]
Die Karten, also die Bilder, kommen aber wie gesagt mit dem "DSA5 System", nicht mit diesem Modul hier.
Wenn du die Karten im "DSA5 System" nicht finden kannst, dann stimmt wohl von Grund auf irgendetwas nicht oder du hast eine uralte DSA5 Version am laufen?

Mach mal eine neue Welt mit nur dem "DSA5 System" und mit dem Modul "Interaktive Aventurien Karte" und schaue ob es dann funktioniert.

Welche Version vom "DSA5 System" läuft bei dir?
Es "muss" die Version 6.1.6+ sein und Version 1.0.7 von diesem Modul hier für Foundry Version 12.

**Comment by Angband666** [comment]
![Screenshot 2024-11-24 161245](https://github.com/user-attachments/assets/14cc5ab5-6c33-4ead-9b8b-800594923dfe)
Einen Schritt weiter bin ich jetzt, ich danke dir. Ich habe die Karte "aventurien_hires" im Modul "dsa5-core" gefunden. Nachdem ich sie manuell in der Szene als Background eingestellt habe, wird sie dann jetzt auch angezeigt. 

Ich wundere mich weiterhin über die Icons, die weiterhin einfache Standard-Icons sind. Möglicherweise hat hier, ebenso wie bei der Karte auch, eine Verlinkung der Grafiken nicht stattgefunden und muss auch manuell erledigt werden? 

Ich nutze Foundry 12 mit DSA Core 6.1.6 sowie Version 1.0.7 von der Karte.

**Comment by JuergenStein** [comment]
Nein, es sollte alles automatisch von selbst funktionieren durch den "Journal Browser" und der Initialisierung von DSA5.
Es muss nichts "manuell" gemacht werden.
Ich habe es gerade mit einer neuen Welt in Foundry 12 und DSA Core 6.1.6 getestet, es hat alles funktioniert und die Karte und die richtigen Icons waren alle da.
